src/Uno.UI/Microsoft/UI/Xaml/Controls/RadioButtons/RadioButtons.cs

```diff
@@ -0,0 +1,740 @@
+﻿using System.Collections.Generic;
+using Microsoft.UI.Private.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Uno.UI.Helpers.WinUI;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	[ContentProperty(Name = nameof(Items))]
+	public partial class RadioButtons : Control
+	{
+		public RadioButtons()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_RadioButtons);
+
+			var items = new List<object>();
```

Comment:
```suggestion
			var items = new ObservableCollection<object>();
```

Reply:
I checked in the WinUI code, it's a `Vector`: `auto const items = winrt::make<Vector<winrt::IInspectable>>();` which are observable.